gemonlinedb/include/gem/onlinedb/GEMOnlineDBManagerWeb.h

```diff
@@ -0,0 +1,45 @@
+/** @file GEMOnlineDBManagerWeb.h */
+
+#ifndef GEM_ONLINEDB_GEMONLINEDBMANAGERWEB_H
+#define GEM_ONLINEDB_GEMONLINEDBMANAGERWEB_H
+
+#include "gem/base/GEMWebApplication.h"
+
+namespace gem {
+  namespace onlinedb {
+
+    class GEMOnlineDBManager;
+
+    class GEMOnlineDBManagerWeb : public gem::base::GEMWebApplication
+    {
+      //friend class GLIBMonitor;
+      friend class GEMOnlineDBManager;
+
+    public:
+      GEMOnlineDBManagerWeb(GEMOnlineDBManager *glibApp);
+
+      virtual ~GEMOnlineDBManagerWeb();
+
+    protected:
+
+      virtual void webDefault(  xgi::Input *in, xgi::Output *out )
+        throw (xgi::exception::Exception);
```

Comment:
`throw()` specifications were deprecated in C++ 11 and removed in C++ 17. Same on the next lines.

Reply:
This cleanup is ongoing in my `xdaq15` migration